Source/Scripts/carthage-zsh-completion

```diff
@@ -18,6 +18,12 @@ __subcommands() {
     _values Commands $commands
 }
 
+__carthage_dependencies() {
+   local -a _dependencies
+   _dependencies=( $(cat Cartfile | grep git | awk -F '"' '{print $2}' | sed -e 's/.*\///' | sed -e 's/\.git$//') )
```

Comment:
This `grep` should probably match only against the beginning of the line so that it doesn't match comments that contain `git`.